tests/integration/iop-integration-test-defaults.yaml

```diff
@@ -45,6 +45,69 @@ spec:
       - name: istio-egressgateway
         enabled: true
   values:
+    sidecarInjectorWebhook:
+      templates:
```

Comment:
small concern: now all installations (ie custom tests) *must* configure this, or all of their tests will start to fail since the pod will fail to be deployed?

Maybe we need SkipGRPC like VM or equivilent?

Reply:
Good point. I'll see if I can make it transparent and just check if the template exists, and skip the deployment.